spec/helpers/faker.rb

```diff
@@ -0,0 +1,12 @@
+# Faker is a top-level module to keep global faker config
+module Faker
```

Comment:
There is a Faker gem. Perhaps we should at least name it something like `MsyncFaker`?

Reply:
Are you OK with ModuleSync::Faker?

Reply:
I think that's ok yes.